docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaSubmissionQueueHandler.php

```diff
@@ -31,7 +33,11 @@ public function postSave(WebformSubmissionInterface $webformSubmission, $update
       $componentAssociatedToWebform = $this->getComponentAssociatedToWebform($webformSubmission);
       if ($componentAssociatedToWebform) {
         $foiaRequest = $this->createFoiaRequest($webformSubmission, $componentAssociatedToWebform);
-        $this->queueFoiaRequest($foiaRequest);
+      }
+
+      if ($foiaRequest && $foiaRequest->getRequestStatus() === 0) {
```

Comment:
Instead of `0`, can we use the constant `FoiaRequestInterface::STATUS_QUEUED` for clarity?

Reply:
Yep. Sounds like a good idea.